ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/LazyQueryResult/LazyQueryResult.tsx

```diff
@@ -320,8 +320,15 @@ export class LazyQueryResult {
     })
     this.syncWithPlain()
   }
-  isDownloadable(): boolean {
-    return this.plain.metacard.properties['resource-download-url'] !== undefined
+  getDownloadUrl(): string {
+    const downloadAction = this.plain.actions.find(
+      (action) =>
+        action.id === 'catalog.data.metacard.resource.deployed-download'
```

Comment:
Realize this isn't really part of this PR but wondering if the action id looked for should be something more generic than "deployed-download" which seems very specific to the downstream use case. Something like `alternate-download` might be a more generic option.

Reply:
Yeah, that had occurred to me too. I'll bring it up in standup and if everyone agrees I'm happy to change it here and put up a PR in the aus repo to update the id.